### DISCUSSION TITLE
Liontron LiFe4Po Batteries

### DISCUSSION THREAD

**Discussion by Linzm99** [Louisvdw/dbus-serialbattery#574] [New BMS requests]
Hi,

I have batteries from Liontron ( https://liontron.com/liontron-lifepo4-speicher/), which are not natively supported by Cerbo GX (but are sold in a Bundle with the Cerbo GX by Greenakku ???). What information do I have to gather for a possible implementation?

Greetings Jan

**Comment by Louisvdw** [comment]
We need the communications protocol for that BMS/battery. If you can get that and upload it to this ticket it might be possible to do.
